File: IOS_Component/18_DataPersitenceCoreData/HJCarModel+CoreDataProperties.m
//
//  HJCarModel+CoreDataProperties.m
//  18_DataPersitenceCoreData
//
//  Created by hongbo ni on 2019/5/28.
//  Copyright © 2019 hongbo ni. All rights reserved.
//
//

#import "HJCarModel+CoreDataProperties.h"

@implementation HJCarModel (CoreDataProperties)

+ (NSFetchRequest<HJCarModel *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"HJCarModel"];
}

@dynamic carName;
@dynamic userId;
@dynamic userName;

@end

File: IOS_Component/18_DataPersitenceCoreData/HJCarModel+CoreDataProperties.swift
//
//  HJCarModel+CoreDataProperties.swift
//  18_DataPersitenceCoreData
//
//  Created by hongbo ni on 2019/5/27.
//  Copyright © 2019 hongbo ni. All rights reserved.
//
//

import Foundation
import CoreData


extension HJCarModel {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<HJCarModel> {
        return NSFetchRequest<HJCarModel>(entityName: "HJCarModel")
    }

    @NSManaged public var carName: String?
    @NSManaged public var userId: Int64
    @NSManaged public var userName: String?

}
